### PR TITLE
Add a multi-material model and JS demo.

### PR DESCRIPTION
**Pull request by prideout**
This is a more complex test than suzanne because it has multiple
materials and exercises the named materials functionality in
loadFilamesh.

This also exercises our glTF-to-Filamesh pipeline at build time.

See issue #663
screen shot 2019-01-09 at 2 11 09 pm

### CODE REVIEW

File: web/samples/helmet.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+<body>
+<canvas></canvas>
+<script src="filament.js"></script>
+<script src="gl-matrix-min.js"></script>
+<script src="https://unpkg.com/gltumble"></script>
+<script>
+
+const ibl_suffix = Filament.getSupportedFormatSuffix('etc s3tc');
+
+const env = 'syferfontein_18d_clear_2k'
```
Comment (bejado) [inline web/samples/helmet.html]:
```suggestion
const env = 'syferfontein_18d_clear_2k';
```

Comment (prideout) [reply web/samples/helmet.html]: Done

**Comment by romainguy** [comment]
Don't forget to move the model to third party with a license file.

**Comment by prideout** [comment]
Good catch, I moved the asset and copied over its accompanying README from the glTF test suite.  

Doing a `git push` from where I am in India is disastrously slow, it is still in progress...

**Pull request merged**